### PR TITLE
Update the Docker quickstart guide

### PR DESCRIPTION
**Pull request by flavio**
Introduce the Docker registry and Portus.

Please read it carefully, my spell checked didn't work against this xml document. Plus I'm not able to render it.

People interested:  @eotchi, @mssola and @taroth21

### CODE REVIEW

**Comment by flavio** [comment]
Just to make it clearn, this is just a starting point. @taroth21 has the final word on it ;)

File: xml/vt_docker_quick.xml [hunk 1]
```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
...
+  <title>Hosting Docker images on-premise</title>
+
+  <para>
+    This section addresses some common questions like: what can be done with
+    the custom Docker images I created, how can they be shared within my
+    organization?
+  </para>
+  <para>
+    The easiest solution would be to push these images to the Docker Hub.
+    However there are a couple of important matters to keep at mind. By default
```
Comment (mssola) [inline xml/vt_docker_quick.xml]: keep at mind -> keep in mind, no ?

File: xml/vt_docker_quick.xml [hunk 2]
```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
    </sect3>
   </sect2>
  </sect1>
+ <sect1 xml:id="sec.docker.registry">
+  <title>Hosting Docker images on-premise</title>
+
+  <para>
+    This section addresses some common questions like: what can be done with
+    the custom Docker images I created, how can they be shared within my
```
Comment (mssola) [inline xml/vt_docker_quick.xml]: I would replace the "," for a "?"

File: xml/vt_docker_quick.xml [hunk 3]
```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
...
+      important to notice that the Docker Hub, as seen by the users, is made
+      at least by the following parts:
+    </para>
+    <itemizedlist mark="bullet" spacing="normal">
+      <listitem>
+        <para>
+          The UI: this is what is actually used by the users with their browser.
+          It provides a nice and intuitive way to browse the contents of the
+          Docker Hub either manually or by using a search feature. It also
+          allosws to create organizations made by different users.
```
Comment (mssola) [inline xml/vt_docker_quick.xml]: typo: allosws -> allows

File: xml/vt_docker_quick.xml [hunk 4]
```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
...
+       <link xlink:href="https://docs.docker.com/registry/"/>.
+     </para>
+  </sect2>
+
+   <sect2 xml:id="sec.docker.registry.limitations">
+     <title>Limitations</title>
+
+     <para>
+       The Docker registry has two major limitations. First of all it lacks
+       any for of authentication. That means everybody with access to the
```
Comment (mssola) [inline xml/vt_docker_quick.xml]: typo: for -> form

File: xml/vt_docker_quick.xml [hunk 5]
```diff
@@ -663,4 +663,163 @@ RUN zypper -n in vim</screen>
...
+       Docker registry can push and pull images to it. That also includes the
+       possibility to even overwrite already existing images.
+     </para>
+
+     <para>
+       There is no way to see which images have been pushed to the Docker
+       registry. You can to manually take note of what is being stored inside
+       of it. There's also no search functionality. This makes collaboration
```
Comment (mssola) [inline xml/vt_docker_quick.xml]: I would re-phrase it like: "There is also no search functionality, which makes collaboration harder."

Comment (flavio) [inline xml/vt_docker_quick.xml]: Fixed, thanks for the tip

**Comment by flavio** [comment]
Thanks @mssola for the tips, I fixed all the issues you reported.

**Comment by mssola** [comment]
LGTM

**Comment by taroth21** [comment]
Many thanks for the pull request and the corrections to it! I will merge it into the development branch and do some minor adjustments with regards to our style guide. 

We are currently creating a maintenance branch for SLE 12 (which will also contain the changes for the Docker update). I'll let you know when we are done with creating the branch and merging the changes.

BTW, you can render the XML sources (and also run a spellcheck etc.) by using DAPS (DocBook Authoring and Publishing Suite). The project's home page is https://github.com/openSUSE/daps. You can find the documentation for it at http://opensuse.github.io/daps/doc/index.html. DAPS is also available as RPM package for openSUSE  and SLE.

**Comment by taroth21** [comment]
@flavio, @mssola , @eotchi: The maintenance branch has been created meanwhile:  https://github.com/SUSE/doc-sle/tree/maintenance/SLE12. I did some more fine-tuning to the file today, mostly cosmetic (in the maintenance branch) . Please have a look at the changes and let me know if you are fine with them. If possible, let me know on Monday if this is the final version that we can publish as an online update for  SLE 12. Thx!

**Comment by mssola** [comment]
It's fine with me.

**Comment by flavio** [comment]
@taroth21 LGTM

**Pull request merged**